.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,92 @@
+name: Bug report
+title: "[Bug] "
+description: Problems and issues with code of Ray
+labels: [bug, triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for finding the time to report the problem! We really appreciate the community efforts to improve Ray.
+        Please make sure what you are reporting is indeed a bug with reproducible steps, if you want to ask questions
+        or share ideas, pleae post on our [Discussion page](https://discuss.ray.io/) instead.
+
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      description: >
+        Please make sure to search in the [issues](https://github.com/ray-project/ray/issues) first to see
+        whether the same issue was reported already.
+      options:
+        - label: >
+            I searched the [issues](https://github.com/ray-project/ray/issues) and found no similar
+            issues.
+          required: true
+
+  - type: dropdown
+    attributes:
+      label: Ray Component
+      description: |
+        What component are you using? Ray has many subprojects, please make sure to choose the component that
+        you found the bug. For third party projects, please refer to their corresponding repos.
+      multiple: true
+      options:
+        - "Ray Core"
+        - "Ray Tune"
+        - "Ray Serve"
+        - "RLlib"
+        - "Ray Clusters"
+        - "Monitoring & Debugging"
+        - "Dashboard"
+        - "RaySGD"
+        - "Others"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: What happened and What you expected to happen
```

Comment:
Description

Reply:
so that it is consistent with the feature request page

Reply:
sorry, I didn't get it about this comment,  the description is under the label.